guides/common/modules/ref_access-to-information-from-insights-in-project.adoc

```diff
@@ -3,7 +3,7 @@
 
 You can access the additional information available for hosts from Red{nbsp}Hat Insights in the following places in the {ProjectWebUI}:
 
-* Navigate to *Configure* > *Insights* where the vertical ellipsis next to the *Remediate* button provides a *View in Red{nbsp}Hat Insights* link to the general recommendations page.
+* Navigate to *Insights* > *Recommendations* where the vertical ellipsis next to the *Remediate* button provides a *View in Red{nbsp}Hat Insights* link to the general recommendations page.
```

Comment:
```suggestion
* Navigate to *Insights* > *Recommendations*, click the vertical ellipsis next to the *Remediate* button, and select *View in Red{nbsp}Hat Insights*.
```
Not sure whether this will help. :-(